#Expyriment 1.0.0 (Python 3.13.7), .xpe-file, coding: cp1252
#date: Wed Sep 24 2025 17:32:21
#sha1: 449ea9
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='zlr778', release='11', version='10.0.22631', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
3613,design,log,
#design: #exp: display-edges
#design: block_cnt,block_id,trial_cnt,trial_id
3613,design,logged,
3613,Experiment,started
3613,stderr,received,'Exception ignored in: <function Visual.__del__ at 0x000001A6BD6EC4A0>'
3613,stderr,received,'Traceback (most recent call last):'
3613,stderr,received,'  File "C:\\Users\\20946\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\expyriment\\stimuli\\_visual.py", line 266, in __del__'
3647,stderr,received,'    if self._compression_filename is not None:'
3649,stderr,received,"AttributeError: 'Circle' object has no attribute '_compression_filename'"
3651,stderr,received,'Traceback (most recent call last):'
3652,stderr,received,'  File "C:\\Users\\20946\\CogSup\\Programming\\Assignments\\Week-3\\Exercises\\display-edges.py", line 14, in <module>\n    rectangle = stimuli.Circle(size = (50, 50), colour = (255, 0, 0), position=(100, 50))'
3653,stderr,received,"TypeError: Circle.__init__() got an unexpected keyword argument 'size'"
